docs/kubernetes/ingress-usage.md

```diff
@@ -178,6 +211,18 @@ Host(/^app-default[.]example[.]org$/) && Method("OPTIONS") ->
   status(200) -> <shunt>
 ```
 
+### Multiple routes
+
+You can also set multiple routes, but you have to set the names of the
+route as defined in eskip:
+
+    zalando.org/skipper-routes: |
+      routename1: Path("/") -> localRatelimit(2, "1h") -> inlineContent("A") -> status(200) -> <shunt>;
+      routename2: Path("/foo") -> localRatelimit(5, "1h") -> inlineContent("B") -> status(200) -> <shunt>
+
+Make sure the `;` semicolon is used to terminate the routes in all,
+but the last route.
```

Comment:
i wonder: in eskip the last semicolon is optional, but it is allowed. In fact, an arbitrary number of semicolons is a valid eskip document. Have we observed any issues with setting a closing semicolon in ingress annotations?